server/conf/application.conf

```diff
@@ -307,6 +307,16 @@ play.filters {
   enabled += filters.HSTSFilter
   enabled += filters.LoggingFilter
   enabled += filters.ValidAccountFilter
+  enabled += play.filters.gzip.GzipFilter
+
+  gzip {
+    threshold = 1000        # don't compress very small responses
+    compressionLevel = -1   # -1 is the default compression level (level 6)
+    contentType {
+      # If non empty, then a response will only be compressed if its content type is in this list.
+      blackList = [ "app/assets/Images/"] 
```

Comment:
Blocking: I don't know that this line does what we want. The docs claim that this should be a MIME Type.

I *believe* you could do this instead:

```
blackList = ["image/*"]
```

Alternatively, we could only gzip well-known types (as in the example in the [docs](https://www.playframework.com/documentation/2.8.x/GzipEncoding#Controlling-which-responses-are-gzipped)). That'd mean we could remove the "blackList" entry and instead add:

```
whiteList = [ "text/*", "application/javascript", "application/json" ]
```

Either way has tradeoffs, so whatever you'd prefer.

Reply:
+1 for only gzipping well-known types

Reply:
Oh shoot, @bion I didn't get to do these yet

Reply:
made https://github.com/seattle-uat/civiform/pull/2667